package.json

```diff
@@ -20,61 +20,65 @@
   "scripts": {
     "audit": "better-npm-audit audit",
     "check-types": "tsc --noEmit",
-    "lint": "eslint ./src/ --ext .ts",
-    "lint:fix": "eslint ./src/ --ext .ts --fix",
+    "lint": "eslint \"src/**/*.ts\"",
+    "lint:fix": "eslint \"src/**/*.ts\" --fix",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
     "test:ci": "vitest run --coverage --reporter junit --outputFile test-results.xml",
     "codegen": "graphql-codegen --config codegen-testing.ts",
     "build": "run-s build:*",
     "build:0-clean": "rimraf dist coverage",
-    "build:1-build": "rollup -c --configPlugin typescript",
-    "build:2-generate-types": "tsc -p tsconfig.json",
-    "build:3-copy-pkg-json": "tstk copy-package-json -c",
-    "build:4-copy-readme": "copyfiles README.md dist"
+    "build:1-lint": "eslint \"src/**/*.ts\" --max-warnings 0",
+    "build:2-check-types": "tsc -p tsconfig.json",
+    "build:3-build": "rollup -c --configPlugin typescript",
+    "build:4-copy-pkg-json": "tstk copy-package-json -c",
+    "build:5-copy-readme": "copyfiles ./README.md ./dist",
+    "format": "prettier --write ."
   },
   "dependencies": {
     "@makerx/graphql-core": "^2.2.2",
-    "@makerx/node-common": "^1.4.0"
+    "@makerx/node-common": "^1.5.0"
   },
   "peerDependencies": {
-    "@apollo/server": "4 || 5",
+    "@apollo/server": "^4.11.2",
```

Comment:
The intention of the peer deps here was to remain as open as possible, so GraphQL consumers can freely change versions without matching them exactly to this library.

Were these specific version changes required or a side-effect of the update?